mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -331,6 +333,14 @@ def handle_rits_export(self) -> None:
             roi = self.view.spectrum_widget.get_roi(ROI_RITS)
             self.model.save_single_rits_spectrum(path, error_mode, roi)
 
+    def handle_export_format_change(self, fmt: str) -> None:
+        self.selected_export_format = fmt
+        LOG.debug(f"Selected export format changed to: {fmt}")
+
+    def handle_export_area_change(self, area: str) -> None:
+        self.selected_export_area = area
+        LOG.debug(f"Selected export area changed to: {area}")
+
```

Comment:
Lets remove these for now. We might not need to do anything until the export button is pressed.